submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/test/security/DeletePermissionTest.kt

```diff
@@ -153,6 +151,39 @@ class DeletePermissionTest(
         assertThat(submissionRepository.getExtByAccNo("DeleteAcc6-2")).isNotNull
     }
 
+    @Test
+    fun `1-7 delete public with regular user and tag access permission`() {
```

Comment:
looks like we need test for delete public by admin which is my understdaning it can be done